packages/artifactory-jcr/package.yaml

```diff
@@ -1,2 +1,2 @@
-url: https://repo.chartcenter.io/jfrog/artifactory-jcr-3.4.0.tgz
+url: https://charts.jfrog.io/artifactory-jcr-3.4.0.tgz
```

Comment:
@PennyScissors can you confirm that this is expected from the partner?

Reply:
> I can't confirm for them, but this will fix our validation which is failing because the old URL gets a 405 Not Allowed, whereas the new pulls the tarball correctly. Seems to be the same chart and I notified them their chart had problems before so I think it should be okay to keep this.

Same thing here. Is basically the same chart, but one doesn't require a license.

Reply:
Confimed with partner this is fine.